configDevoted.yml

```diff
@@ -127,396 +146,1827 @@ recipes:
+      stone:
+        material: SANDSTONE
+        amount: 256
+    output:
+      poweredrails:
+        material: POWERED_RAIL
+        amount: 24
+  Stone Obsidian:
+    type: PRODUCTION
+    name: Stone Obsidian
+    production_time: 5s
     input:
-      melon:
-        material: MELON
-        amount: 1
-      stn:
-        material: STONE
-        amount: 1
-      irn:
+      coal:
+        material: COAL
+        amount: 16
+      iron:
         material: IRON_INGOT
-        amount: 1
-      dbred:
-        material: REDSTONE_BLOCK
-        amount: 1
-      diams:
+        amount: 16
+      lapis:
+        material: INK_SACK
+        durability: 4
+        amount: 16
+      gold:
+        material: GOLD_INGOT
+        amount: 16
+      redstone:
+        material: REDSTONE
+        amount: 16
+      diamond:
         material: DIAMOND
-        amount: 1
-      emrd:
+        amount: 16
+      emerald:
         material: EMERALD
-        amount: 1
-    factory: Material Synthesizer
-  rec16:
-    name: Upgrade to Enhanced Smelter
-    production_time: 20s
-    type: UPGRADE
+        amount: 16
+    output:
+      Stone_Obsidian:
+        material: OBSIDIAN
+        amount: 128
+        lore:
+        - Stone Reinforced Obsidian
+  Iron Obsidian:
+    type: PRODUCTION
+    name: Iron Obsidian
+    production_time: 5s
     input:
-      m1:
-        material: IRON_BLOCK
-        amount: 15
-      m2:
+      lead:
+        material: COAL
+        amount: 8
+        lore:
+        - Lead T1
+      steel:
+        material: IRON_INGOT
+        amount: 8
+        lore:
+        - Steel T1
+      bronze:
+        material: INK_SACK
+        durability: 4
+        amount: 4
+        lore:
+        - Bronze T1
+      silver:
+        material: GOLD_INGOT
+        amount: 4
+        lore:
+        - Silver T1
+      copper:
+        material: REDSTONE
+        amount: 4
+        lore:
+        - Copper T1
+      titanium:
         material: DIAMOND
-        amount: 64
-      m3:
+        amount: 4
+        lore:
+        - Titanium T1
+      mercury:
         material: EMERALD
-        amount: 10
-      m4:
-        material: LAPIS_BLOCK
-        amount: 15
-      m5:
-        material: COAL_ORE
-        amount: 640
-    factory: Enhanced Smelter
-  rec17:
-    name: Sift Diamonds
-    production_time: 3s
-    type: PRODUCTION
-    input:
-      d1:
-        material: DIAMOND_ORE
-        amount: 2
+        amount: 4
+        lore:
+        - Mercury T1
     output:
-      d2:
-        material: DIAMOND
-        amount: 5
-  rec18:
-    name: Extract Gems
-    production_time: 3s
+      Iron_Obsidian:
+        material: OBSIDIAN
+        amount: 16
+        lore:
+        - Iron Reinforced Obsidian
+  Diamond Obsidian:
     type: PRODUCTION
+    name: Diamond Obsidian
+    production_time: 5s
     input:
-      g1:
-        material: EMERALD_ORE
-        amount: 2
-    output:
-      g2:
+      nickel:
+        material: COAL
+        amount: 64
```

Comment:
I think Bonkill is planning on having display names all throughout.

To give an item a custom name, use the `name:` tag. I'll do the same in HiddenOre.